litellm/model_prices_and_context_window_backup.json

```diff
@@ -1094,6 +1094,14 @@
         "litellm_provider": "vertex_ai-anthropic_models",
         "mode": "chat"
     },
+    "vertex_ai/claude-3-opus@20240229": {
+        "max_tokens": 200000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.000015,
+        "output_cost_per_token": 0.000075,
+        "litellm_provider": "vertex_ai-anthropic_models",
+        "mode": "chat"
```

Comment:
@Manouchehri do you mind adding source, so we know where the pricing is from? 

For vertex i always find it hard to navigate their docs and grab the pricing